server/broadcast.go

```diff
@@ -346,12 +356,28 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string,
 
 	// send segment to the orchestrator
 	glog.V(common.DEBUG).Infof("Submitting segment nonce=%d manifestID=%s seqNo=%d orch=%s", nonce, cxn.mid, seg.SeqNo, sess.OrchestratorInfo.Transcoder)
+	if sess.Sender != nil {
+		err := sess.Sender.ValidateTicketParams(pmTicketParams(sess.OrchestratorInfo.TicketParams))
+		if err != nil {
+			if err != pm.ErrTicketParamsExpired {
+				return nil, err
```

Comment:
I think we'll need to call `cxn.sessManager.removeSession()` - if the error is not an expiry error then we just remove the O.

Reply:
fixed in 20340ceabc68452fb4c8914a568e965e1c5f5d30